src/kbmod/analysis/plotting.py

```diff
@@ -0,0 +1,382 @@
+        An `(x, y)` pair of coordiantes
+    radius : `float`
+        Radius of the circle around the center.
+
+    Returns
+    -------
+    ax : `matplotlib.pyplot.Axes`
+        Modified Axis.
+    """
+    ax.scatter(*center, color="black", label="Pointing area")
+    circ = plt.Circle(center, radius, fill=False, color="black")
+    ax.add_artist(circ)
+    return ax
+
+
+def plot_bbox(ax, bbox):
+    """Adds the footprint defined by the given BBOX to the axis.
+
+    Parameters
+    ----------
+    ax : `matplotlib.pyplot.Axes`
+        Axis.
+    bbox : `list`
+        List of 4 tuples representing (x, y) coordinates of the
+        corners of a rectanlge, in clockwise convention.
+
+    Returns
+    -------
+    ax : `matplotlib.pyplot.Axes`
+        Modified Axis.
+    """
+    xy, width, height, angle = transform_rect(bbox)
+    rect = plt.Rectangle(xy, width, height, angle=angle, fill=None, color="black")
+    ax.add_artist(rect)
+    return ax
+
+
+def plot_bboxes(ax, bboxes):
+    """Adds the footprints defined by each given BBOX to the axis.
+
+    Parameters
+    ----------
+    ax : `matplotlib.pyplot.Axes`
+        Axis.
+    bbox : `list[list]`
+        List of bboxes. Each bbox is a list of 4 tuples representing
+        (x, y) coordinates of the corners of a rectanlge, in clockwise
+        convention.
+
+    Returns
+    -------
+    ax : `matplotlib.pyplot.Axes`
+        Modified Axis.
+    """
+    for bbox in bboxes:
+        ax = plot_bbox(ax, bbox, figure)
+    return ax
+
+
+def plot_footprint(ax, wcs):
+    """Adds the footprint defined by the given WCS to the axis.
+
+    Parameters
+    ----------
+    ax : `matplotlib.pyplot.Axes`
+        Axis.
+    wcs : `astropy.wcs.WCS`
+        World Coordinate System instance from which the footprint will
+        be calculated from.
+
+    Returns
+    -------
+    ax : `matplotlib.pyplot.Axes`
+        Modified Axis.
+    """
+    xy, width, height, angle = transform_rect(wcs.calc_footprint())
+    rect = plt.Rectangle(xy, width, height, angle=angle, fill=None, color="black")
+    ax.add_artist(rect)
+    return ax
+
+
+def plot_footprints(ax, wcs_list):
+    """Adds the footprints defined by each given WCS to the axis.
+
+    Parameters
+    ----------
+    ax : `matplotlib.pyplot.Axes`
+        Axis.
+    wcs_list : `list[astropy.wcs.WCS]`
+        List of WCS objects whose footprints are being plotted.
+
+    Returns
+    -------
+    ax : `matplotlib.pyplot.Axes`
+        Modified Axis.
+    """
+    for wcs in wcs_list:
+        ax = plot_footprint(ax, wcs, figure)
+    return ax
+
+
+def plot_all_objs(ax, objects, count=0, show_field=False, center=None, radius=1.1, lw=0.9, ms=1):
+    """Plots object tracks on the given axis.
+
+    Parameters
+    ----------
+    ax : `matplotlib.pyplot.Axes`
+        Axis.
+    objects : `astropy.table.Table`
+        Table of objects. Must contain column ``Name`` identifying
+        unique objects.
+    count : `int`
```

Comment:
You should mention that if count == 0 then it plots all of the objects. I figured it would plot zero until I got to the code below.

Reply:
I made that `-1` to make it less confusing (ironically)